detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/rulesdetekt/wrapper/Executable.kt

```diff
@@ -0,0 +1,37 @@
+package io.buildfoundation.bazel.rulesdetekt.wrapper
+
+import java.io.File
+import java.io.FileOutputStream
+import java.io.PrintStream
+import java.nio.charset.Charset
+
+internal interface Executable {
+
+    fun execute(args: Array<String>): Result
+
+    class DetektImpl(private val detekt: Detekt) : Executable {
+
+        override fun execute(args: Array<String>): Result {
+            val outputPrinter = PrintStream(Streams.DevNullOutputStream().buffered())
```

Comment:
Hold on, you don't want to capture stdout? Detekt prints quite a lot of useful stuff there

Reply:
Or I guess this is due to multithreading?

Reply:
There might be a way to achieve normal stdout with multithreading, but that'll require ClassLoader based isolation unless we patch Detekt…

Reply:
I’ve actually extended the Detekt API to print everything to streams we provide, not system ones (see the attached issue). But yeah, I ignore the output since it prints a console report all the time — no matter the execution success or failure. It actually doesn’t point where the error happened — just lists everything detected (detekted?). I’ve assumed that file-based reports should be used to determine what actually went wrong. The official Gradle plugin behaves this way — error message and no output.